home-data-migration/src/main/resources/db/changelog/0.0.1/Issue134.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="Issue134.001" author="alexorlenko">
+        <createTable tableName="contact">
```

Comment:
may be add  <preConditions onFail="MARK_RAN">
            <not>
                <tableExists tableName="contact"/>
            </not>
        </preConditions>